timestampvm/service.go

```diff
@@ -44,14 +45,6 @@ func (s *Service) ProposeBlock(_ *http.Request, args *ProposeBlockArgs, reply *P
 	return nil
 }
 
-// APIBlock is the API representation of a block
-type APIBlock struct {
-	Timestamp json.Uint64 `json:"timestamp"` // Timestamp of most recent block
-	Data      string      `json:"data"`      // Data in the most recent block. Base 58 repr. of 5 bytes.
-	ID        string      `json:"id"`        // String repr. of ID of the most recent block
-	ParentID  string      `json:"parentID"`  // String repr. of ID of the most recent block's parent
-}
-
 // GetBlockArgs are the arguments to GetBlock
 type GetBlockArgs struct {
```

Comment:
nit: can we add a json tag here for `ID`?

Reply:
If we change `ID`'s type to `ids.ID` then we can just have the marshaller deal with the conversion.